packages/apps/kadena-docs/src/scripts/remarkFrontmatterToProps.js

```diff
@@ -8,16 +9,30 @@ const getFrontMatter = (node) => {
   }
 };
 
+const getModifiedDate = (file) => {
+  const stats = fs.statSync(file);
+  if (!stats.isFile()) return;
+  return stats.mtimeMs;
+};
+
+const getFileName = (file) => {
+  if (file.history.length === 0) return '';
+  return file.history[0];
+};
```

Comment:
I prefer to use functions when they're like this. It makes it easier to debug as the function name is shown in the stack trace